### PR TITLE
Update dodo.py

### PR DESCRIPTION
**Pull request by boyhominid**
### Pull Request Checklist

- [X] I’ve read the [guidelines for contributing](https://github.com/getnikola/nikola/blob/master/CONTRIBUTING.rst).
- [X] I updated AUTHORS.txt and CHANGES.txt (if the change is non-trivial) and documentation (if applicable).
- [X] I tested my changes.

### Description
Added script annotations to functions

### CODE REVIEW

File: dodo.py [hunk 1]
```diff
@@ -8,45 +8,45 @@
 }
 
 
-def recursive_glob(path, pattern):
+def recursive_glob(path, pattern) -> str:
```
Comment (Kwpolska) [inline dodo.py:11]: This function does not return a single string.

File: dodo.py [hunk 2]
```diff
@@ -8,45 +8,45 @@
...
     """recursively walk path directories and return files matching the pattern"""
     for root, dirnames, filenames in os.walk(path, followlinks=True):
         for filename in fnmatch.filter(filenames, pattern):
             yield os.path.join(root, filename)
 
 
-def task_flake8():
+def task_flake8() -> str:
```
Comment (Kwpolska) [inline dodo.py:18]: This function does not return a string.

File: dodo.py [hunk 3]
```diff
@@ -8,45 +8,45 @@
...
     """flake8 - static check for python files"""
     yield {
         'name': os.path.join(os.getcwd(), 'nikola'),
         'actions': ['flake8 nikola/'],
     }
 
 
-def task_pydocstyle():
+def task_pydocstyle() -> str:
```
Comment (Kwpolska) [inline dodo.py:26]: This function does not return a string.

File: dodo.py [hunk 4]
```diff
@@ -8,45 +8,45 @@
...
     """pydocstyle -- static check for docstring style"""
     yield {
         'name': os.path.join(os.getcwd(), 'nikola'),
         'actions': ["pydocstyle --count --match-dir='(?!^\\.)(?!data).*' nikola/"],
     }
 
 
-def task_test():
+def task_test() -> str:
```
Comment (Kwpolska) [inline dodo.py:34]: This function does not return a string.

File: dodo.py [hunk 5]
```diff
@@ -8,45 +8,45 @@
...
     """run unit-tests using py.test"""
     return {
         'actions': ['py.test tests/'],
     }
 
 
-def task_coverage():
+def task_coverage() -> str:
```
Comment (Kwpolska) [inline dodo.py:41]: This function does not return a string.

File: dodo.py [hunk 6]
```diff
@@ -8,45 +8,45 @@
...
     """run unit-tests using py.test, with coverage reporting"""
     return {
         'actions': ['py.test --cov nikola --cov-report term-missing tests/'],
-        'verbosity': 2,
+        'verbosity' int: 2,
     }
 
 
-def task_gen_completion():
+def task_gen_completion() -> str:
```
Comment (Kwpolska) [inline dodo.py:49]: This function does not return a string.

File: dodo.py [hunk 7]
```diff
@@ -8,45 +8,45 @@
...
     }
 
 
-def task_coverage():
+def task_coverage() -> str:
     """run unit-tests using py.test, with coverage reporting"""
     return {
         'actions': ['py.test --cov nikola --cov-report term-missing tests/'],
-        'verbosity': 2,
+        'verbosity' int: 2,
```
Comment (Kwpolska) [inline dodo.py:45]: This is invalid syntax.

**Review by Kwpolska — CHANGES_REQUESTED** [review]

**Comment by boyhominid** [comment]
Super apologetic here, just getting started and I want to help. I'll take a look at things before I re-submit changes you've requested

**Comment by Kwpolska** [comment]
It is OK to make mistakes. It is also important to test your changes before opening a pull request. For type annotations, this would mean checking the code with `mypy` or `pyright`. I would also recommend focusing on the main codebase in the `nikola` folder.

**Pull request closed without merging**